pkgs/tools/graphics/scrot/default.nix

```
@@ -1,19 +1,20 @@
 { lib, stdenv, fetchFromGitHub, giblib, xlibsWrapper, autoreconfHook
-, autoconf-archive, libXfixes, libXcursor, libXcomposite }:
+, autoconf-archive, libXfixes, libXcursor, libXcomposite
+, pkg-config, gettext, libtool, intltool, gtk-doc, libbsd }:
 
 stdenv.mkDerivation rec {
   pname = "scrot";
-  version = "1.5";
+  version = "1.7";
 
   src = fetchFromGitHub {
     owner = "resurrecting-open-source-projects";
     repo = pname;
     rev = version;
-    sha256 = "sha256-4vguodLnCj0sOBLM4oJXTfX1p8hIo3WTwIuViPtZxHQ=";
+    sha256 = "sha256-oVmEPkEK1xDcIRUQjCp6CKf+aKnnVe3L7aRTdSsCmmY=";
   };
 
-  nativeBuildInputs = [ autoreconfHook autoconf-archive ];
-  buildInputs = [ giblib xlibsWrapper libXfixes libXcursor libXcomposite ];
+  nativeBuildInputs = [ autoreconfHook autoconf-archive pkg-config ];
+  buildInputs = [ giblib xlibsWrapper libXfixes libXcursor libXcomposite gettext libtool intltool gtk-doc libbsd ];
```

```suggestion
  buildInputs = [ giblib xlibsWrapper libXfixes libXcomposite gettext libtool intltool gtk-doc libbsd ];
```
libXcursor is not required since 1.4.

---

Dankeschön!